addon/components/base/bs-button.js

```diff
@@ -485,13 +460,12 @@ export default class Button extends Component {
           `After:\n` +
           `  <BsButton ${typeof value === 'boolean' ? attribute : `${attribute}="${value}"`} />`;
 
-        deprecate(
-          deprecationMessage,
+        warn(
```

Comment:
Just asking myself if we shouldn't throw/assert here rather? The warning can be overseen, and I wouldn't expect people to have such rigid test coverage to detect this kind of missing attributes...

Reply:
I also considered throwing instead of warning but was afraid that in some weird edge cases a consumer likes to pass that arguments even so it's fine if they don't have any affect. E.g. for dynamic component invocation by `{{component}}` helper and some of the components require an argument with that exact name. Not sure so how realistic such a scenario is.